spec/controllers/concerns/transaction/current_spec.rb

```diff
@@ -19,7 +19,7 @@ def index
 	end
 
 	it 'gets transaction if found' do
-		get :index, { nonprofit_id: nonprofit.houid, id: transaction.houid }
+		get :index, params: { nonprofit_id: nonprofit.houid, id: transaction.houid }
```

Comment:
Update for deprecation of positional params in calls in controller and request tests